pages/atoms/carousel-next-button/carousel-next-button.tsx

```diff
@@ -4,7 +4,11 @@ import { CarouselButton } from "../../../data";
 
 export default function CarouselNextButton(props: CarouselButton) {
   return (
-    <button className={style.carouselButton} onClick={props.loadNextSlide}>
+    <button
+      className={style.carouselButton}
+      onClick={props.loadNextSlide}
+      data-testid="buttonNext"
```

Comment:
rule of thumb is to use data-testid as last resort, you could add aria-label e.g. 'previous' and then use getByRole('button', {name: 'previous'} )
also afaik for buttons, the aria label / button name is determined by the text content between the button tags, this case there is image which doesnt have any text, using getByRole would solve this accessibility issue too